tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -560,6 +582,321 @@ def _(
+        except Exception as e:
+            logger.debug(f"CuteDSL get_valid_tactics failed: {e}")
+            return []
+
+    def forward(
+        self,
+        inputs: List[torch.Tensor],
+        tactic=-1,
+    ) -> torch.Tensor:
+        """Forward pass through CuteDSL runner."""
+        # CuteDSL only needs first 4 inputs (alpha is already in the runner)
+        act_fp4, weight, act_sf, weight_scale, alpha = inputs
```

Ruff (0.14.3)

Unpacked variable `alpha` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)